build/run-tests.sh

```diff
@@ -1,7 +1,40 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
+set -e
 
-docker-compose build --no-cache
+SCRIPT_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
+
+function stop_containers() {
+  docker-compose down
+}
+
+trap stop_containers EXIT
+
+if [[ -z "${VERIFY_EVENT_SYSTEM_DATABASE_SCRIPTS_LOCATION}" ]]; then
+    export VERIFY_EVENT_SYSTEM_DATABASE_SCRIPTS_LOCATION=${SCRIPT_DIR}/../../verify-event-system-database-scripts
+fi
+
+echo Database Scripts are located at: ${VERIFY_EVENT_SYSTEM_DATABASE_SCRIPTS_LOCATION}
+
+if [[ ! -d "${VERIFY_EVENT_SYSTEM_DATABASE_SCRIPTS_LOCATION}" ]]; then
+    echo "Cloning database migration scripts repo..."
+    pushd ${SCRIPT_DIR}/../..
```

Comment:
So if you *have* set `VERIFY_EVENT_SYSTEM_DATABASE_SCRIPTS_LOCATION` _but it doesn't exist_ then this doesn't quite work! Something like the one I sent in my earlier comment might be right. Ho hum :-D